ApplyToBecomeInternal/ApplyToBecomeInternal/Models/DateInputModelBinder.cs

```diff
@@ -58,10 +68,30 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
 				bindingContext.ModelState.SetModelValue(monthModelName, monthValueProviderResult);
 				bindingContext.ModelState.SetModelValue(yearModelName, yearValueProviderResult);
 
-				bindingContext.ModelState.TryAddModelError(
-					bindingContext.ModelName,
-					new FormatException("Invalid date specified."),
-					bindingContext.ModelMetadata);
+				if (_missingParts.Count == 3)
```

Comment:
I might be missing a requirement here but is it not a valid usecase that the user might want to enter the date later but update other fields now e.g. on the local authority page they could want to add comments and link but do the dates later?

Reply:
Ok its optional now